error_test.go

```diff
@@ -61,6 +61,13 @@ func TestErrors(t *testing.T) {
 			errText:  "App '/not_existent' does not exist",
 			content:  `{"message": "App '/not_existent' does not exist"}`,
 		},
+		// 405
+		{
+			httpCode: http.StatusMethodNotAllowed,
+			errCode:  ErrCodeMethodNotAllowed,
+			errText:  "",
+			content:  `{"message": null}`,
```

Comment:
Did you possibly verify that Marathon returns no real content? At least at the time when we created the various error codes, it usually did.

Reply:
yep, that null thing inside json is what marathon returns when you do HTTP DELETE on /v2/apps

Reply:
Gotcha, thanks. 👍